### PR TITLE
Test Cases for ChainManager Module

### PR DESCRIPTION
**Pull request by murthyvitwit**
#477

### CODE REVIEW

File: x/chainmanager/common_test.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package chainmanager_test
```
Comment (ssandeep) [inline x/chainmanager/common_test.go:1]: How about moving this to test_helper.go (similar to what you did in Checkpoint module)?

**Review by ssandeep — CHANGES_REQUESTED** [review]

File: x/chainmanager/common_test.go (hunk 1 above)
Comment (ironman0x1967) [reply x/chainmanager/common_test.go:1]: Okay

**Comment by ironman0x1967** [comment]
@ssandeep review changes are addressed and i tried to minimize the line issue but some lint issues are related to gov module 
if i change here after some time gov test cases will fail i think 

```
x/gov/types/deposit.go:56:36        ineffassign  ineffectual assignment to `v`
x/gov/types/vote.go:76:33           ineffassign  ineffectual assignment to `v`
```

and genrandomaccounts lint issue we are fixing the staking test cases branch 
```
x/staking/simulation/utils.go:27:5  ineffassign  ineffectual assignment to `startBlock`
x/staking/simulation/utils.go:31:5  ineffassign  ineffectual assignment to `power`
```

**Review by ssandeep — APPROVED** [review]
OK, got it.

**Pull request merged**